src/data-source.ts

```diff
@@ -20,4 +13,25 @@ export const AppDataSource = new DataSource({
   entities: ['src/entity/**/*.{js,ts}'],
   migrations: ['src/migration/**/*.{js,ts}'],
   subscribers: ['src/subscriber/**/*.{js,ts}'],
+  replication: {
+    defaultMode: 'master',
+    master: {
+      host: process.env.TYPEORM_HOST || 'localhost',
+      port: 5432,
+      username: process.env.TYPEORM_USERNAME || 'postgres',
+      password: process.env.TYPEORM_PASSWORD || '12345',
+      database:
+        process.env.TYPEORM_DATABASE ||
+        (process.env.NODE_ENV === 'test' ? 'api_test' : 'api'),
+    },
+    slaves: [
+      {
+        host: process.env.TYPEORM_READ_HOST || 'localhost',
+        port: 5432,
+        username: process.env.TYPEORM_READ_USERNAME || 'postgres',
+        password: process.env.TYPEORM_READ_PASSWORD || '12345',
+        database: process.env.TYPEORM_READ_DATABASE,
```

Comment:
Fallback for adhoc?

Reply:
We can set fallback to test db? (we need some fallback else it will fail to spool.
Let me know what you think?

Reply:
Just specify master as slave in adhoc (so we can still use `slave` in query build?

Reply:
(to avoid having another db locally as you said)

Reply:
Yeah that also works :)